src/MSALWrapper/AuthFlow/IntegratedWindowsAuthentication.cs

```diff
@@ -72,7 +72,7 @@ protected override async Task<TokenResult> GetTokenInnerAsync()
             }
             catch (MsalClientException ex) when (ex.Message.Contains("WS-Trust endpoint not found"))
             {
-                this.logger.LogWarning($"IWA only works on Corp Net, please turn on VPN.");
+                this.logger.LogInformation($"IWA only works on Corp Net, please turn on VPN.");

```

Comment:
Related to #178, should we also consider changing the wording too?

Reply:
I think we can be more specific, and provide a clear instruction